csrc/angle/harmonic_angle_interface.cpp

```diff
@@ -1,7 +1,7 @@
 #include <pybind11/pybind11.h>
 #include <torch/library.h>
 
-TORCH_LIBRARY(torchff, m) {
+TORCH_LIBRARY_FRAGMENT(torchff, m) {
```

Comment:
I asked GPT and it seems that here is the reason to cause the error. We already have a `TORCH_LIBRARY` in harmonic bond under namespace `torchff`, so here it needs to be changed into `TORCH_LIBRARY_FRAGMENT` .

GPT also suggested another way to fix this, which is to put all the `TORCH_LIBRARY` macro in one `interface.cpp` file. But I thought that will change the code structure too much and make it less modular.